rocAL/rocAL_pybind/amd/rocal/plugin/tf.py

```diff
@@ -107,17 +107,17 @@ def __next__(self):
             self.num_bboxes_list=[]
             #Count of labels/ bboxes in a batch
             self.bboxes_label_count = np.zeros(self.bs, dtype="int32")
-            self.count_batch = self.loader.GetBoundingBoxCount(self.bboxes_label_count)
+            self.count_batch = self.loader.getBoundingBoxCount(self.bboxes_label_count)
             self.num_bboxes_list = self.bboxes_label_count.tolist()
             # 1D labels array in a batch
             self.labels = np.zeros(self.count_batch, dtype="int32")
-            self.loader.GetBBLabels(self.labels)
+            self.loader.getBBLabels(self.labels)
             # 1D bboxes array in a batch
             self.bboxes = np.zeros((self.count_batch*4), dtype="float32")
-            self.loader.GetBBCords(self.bboxes)
+            self.loader.getBBCords(self.bboxes)
```

Comment:
There is api called getBoundingBoxLabels, getBoundingBoxCoords
Please check and see if we can use single api itself

Reply:
The first one returns a np array of int32 dtype while the second one returns a np array of float32. Hence we are using two APIs to get the labels and coords

Reply:
I think this is not a good practice, please check and merge them into single api if possible